mungegithub/mungers/mungerutil/util.go

```diff
@@ -66,6 +76,20 @@ func (u UserSet) Mention() UserSet {
 	return UserSet(mentionedUsers)
 }
 
+func (u UserSet) UnMention() UserSet {
```

Comment:
Depends how we implement the OWNERS file, let's not merge until we actually need it ...